GTG/gtk/browser/browser.py

```diff
@@ -210,22 +195,6 @@ def init_tags_sidebar(self):
         # expanding search tag does not work automatically, request it
         self.expand_search_tag()
 
-    def _init_toolbar_tooltips(self):
-        """
-        sets tooltips for widgets on toolbars
-        """
-        self.donebutton.set_tooltip_text(GnomeConfig.MARK_DONE_TOOLTIP)
-        self.editbutton.set_tooltip_text(GnomeConfig.EDIT_TOOLTIP)
-        self.dismissbutton.set_tooltip_text(GnomeConfig.MARK_DISMISS_TOOLTIP)
-        self.newtask.set_tooltip_text(GnomeConfig.NEW_TASK_TOOLTIP)
-        self.newsubtask.set_tooltip_text(GnomeConfig.NEW_SUBTASK_TOOLTIP)
-        self.toggle_workview.set_tooltip_text(
-            GnomeConfig.WORKVIEW_TOGGLE_TOOLTIP)
-        self.quickadd_entry.set_tooltip_text(
```

Comment:
Can you bring back this tooltip and set it directly in taskbrowser.ui? I don't think it's dynamically set anywhere.